### PR TITLE
Add pyvo meetup

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

**Comment by hypha** [comment]
@clytaemnestra I am not sure where the RSVP button should point to. Please feel free to change anything! 

For the blurb, I tried to make the EuroPython mentions in self-contained sentences, so if you need to have copies for Czech, you can also remove those sentences. But again, review and suggest!

**Comment by clytaemnestra** [comment]
> @clytaemnestra I am not sure where the RSVP button should point to. Please feel free to change anything!
> 
> For the blurb, I tried to make the EuroPython mentions in self-contained sentences, so if you need to have copies for Czech, you can also remove those sentences. But again, review and suggest!

Are you sure you've committed everything? I see only link to the page, but the page doesn't exist.

[Here](https://www.meetup.com/pyvo-prague-python-meetup/events/nzltctyfckbzb/) 's the link to the meetup. I'll then update it afterwards and let you know before we merge this one, so both would go live at the same time 🙂

**Comment by hypha** [comment]
@clytaemnestra I am such an idiot. The files are added and you can preview them. After we fix are the content, I'll merge when you tell me to merge.

File: data/pages-content/pyvo-meetup.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---  
+
+# Pyvo Meetup
+## About Pyvo
+ “Pivo” means “beer” in Czech, and Pyvo is a meetup for fans of the Python programming language and related technologies. These meetups are held in several cities across the country. The meetups are quite informal, and they're held in pubs usually with talks and mingling.
+## About the Event
+Prague Pyvo is traditionally held on the third Wednesday of each month. The July edition will take place during EuroPython! Join us for our July Pyvo meetup in Prague, where we'll be hosting a delightful grill & chill session. We invite you to connect with the Czech local Python community in Prague, enjoy the perfect summer vibes of Prague and exchange knowledge and ideas!
+
+<div style={{textAlign: "center", marginBottom: 20}}>
+<ButtonLink href="https://www.meetup.com/pyvo-prague-python-meetup/events/nzltctyfckbzb/"> Register your interest now! </ButtonLink>
```
Comment (clytaemnestra) [inline data/pages-content/pyvo-meetup.md:15]:
```suggestion
<ButtonLink href="https://www.meetup.com/pyvo-prague-python-meetup/events/293781460/"> Register your interest now! </ButtonLink>
```

Comment (clytaemnestra) [reply data/pages-content/pyvo-meetup.md:15]: Looks great, thanks a lot! 🙂 I've sent you the wrong link, so I've added one suggestion. I've updated our web and meetup, so I guess we can merge it? 🙂

**Pull request merged**